test/DurableOrchestrationClientMock.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using DurableTask.Core;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Newtonsoft.Json;
+
+namespace WebJobs.Extensions.DurableTask.Tests
+{
+    public class DurableOrchestrationClientMock : DurableOrchestrationClient
+    {
+
```

Comment:
nit: can we remove all these extra newlines?

Reply:
Sure